runtime/lua/vim/lsp/inlay_hint.lua

```diff
@@ -130,9 +115,17 @@ local function clear(bufnr)
   api.nvim__buf_redraw_range(bufnr, 0, -1)
 end
 
-local function make_request(request_bufnr)
-  reset_timer(request_bufnr)
-  util._refresh('textDocument/inlayHint', { bufnr = request_bufnr })
+--- Disable inlay hints for a buffer
+---@param bufnr (integer) Buffer handle, or 0 for current
+local function disable(bufnr)
+  if bufnr == nil or bufnr == 0 then
```

Comment:
there has `resolve_bufnr` local  function

Reply:
That function is local to `lsp.lua`, though, and duplicated (in a simplified form) in `codelens.lua`. Could be refactored to a (`@private`) `util.lua` function in a followup PR.

Reply:
nope. `inlay_hint.lua` also have this function see line 93

Reply:
Not in my repo. Link please?

Reply:
hmm okay.. my local repo outdate.  it refactored by mfussenegger .